app/controllers/general.php

```diff
@@ -847,6 +847,10 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
                 \var_dump('Wrapping authorization exception');
                 $error = new AppwriteException(AppwriteException::USER_UNAUTHORIZED);
                 break;
+            case 'Utopia\Database\Exception\InvalidRelationshipValue':
+                \var_dump('Wrapping relationship exception');
+                $error = new AppwriteException(AppwriteException::RELATIONSHIP_VALUE_INVALID);
```

Comment:
Let's pass the message from the original exception into the appwrite exception too

Reply:
Done!